test/GithubApi.Gist.test.js

```diff
@@ -0,0 +1,44 @@
+const { expect } = require('chai');
+const axios = require('axios');
+const chaiSubset = require('chai-subset');
+const chai = require('chai');
+
+chai.use(chaiSubset);
+/* Instances from the axios class that has headers attribute
+ to do the authentication */
+const object = axios.create({
+  headers: {
+    Authorization: `token ${process.env.ACCESS_TOKEN}`
+  }
+});
+
+const body = { description: 'this is a example about promises', public: true, files: { 'README.md': { content: 'this is a example about promises' } } };
```

Comment:
split this line into multiple lines to improve the readability 
```suggestion
const body = {
  description: 'this is a example about promises',
  public: true,
  files: {
    'README.md': {
      content: 'this is a example about promises'
    }
  }
};
```